util/EIP-1024/src/lib.rs

```diff
@@ -0,0 +1,139 @@
+extern crate saltbabe;
+#[macro_use]
```

Comment:
We use `tabs` across the entire codebase, could you please reformat this file (and all other files) accordingly?

Reply:
yes, printing is a bad idea indeed. I just put that for debugging.

Reply:
ok